packages/engine/crawlers/sui/subscription.ts

```diff
@@ -22,6 +25,19 @@ export const suiTokenSubscribe = (
 	);
 };
 
+export const suiCollectibleSubscribe = async (
+	connection: JsonRpcProvider,
+	owner: string,
+) => {
+	suiSubscriptionPool.push(
+		setInterval(async () => {
+			const collectibleDocs = await getSuiCollectibles(connection, owner);
+
+			collectibleActions.setCollectibles(collectibleDocs);
+		}, 30000),
+	);
```

Comment:
mark to todo comment to optimize it later I think

Reply:
Sui has not support onChange events, already checked with Sui Foundation and explored Ethos, Suiet wallets, so this is the possible solution right now